src/components/dialogs/filter/expert/expert-filter-constants.ts

```diff
@@ -227,4 +247,17 @@ export const fields: Record<string, Field[]> = {
         FIELDS_OPTIONS.VOLTAGE_LEVEL_ID,
     ],
     LOAD: [FIELDS_OPTIONS.ID],
+    VOLTAGE_LEVEL: [
+        FIELDS_OPTIONS.ID,
+        FIELDS_OPTIONS.NAME,
+        FIELDS_OPTIONS.NOMINAL_VOLTAGE,
+        FIELDS_OPTIONS.LOW_VOLTAGE_LIMIT,
+        FIELDS_OPTIONS.HIGH_VOLTAGE_LIMIT,
+        FIELDS_OPTIONS.COUNTRY,
+    ],
+    SUBSTATIONS: [
```

Comment:
```suggestion
    SUBSTATION: [
```
typo ? Otherwise fields don't show up when selecting substations

Reply:
Ah yes my bad thanks